model/clusters_mgmt/v1/azure_type.model

```diff
@@ -101,4 +101,8 @@ struct Azure {
 	// Required during creation.
 	// Immutable.
 	OperatorsAuthentication AzureOperatorsAuthentication
+
+	// NodesOutboundConnectivity defines how the network outbound
+	// configuration of the Cluster's Node Pool's Nodes is performed.
+	NodesOutboundConnectivity AzureNodesOutboundConnectivity
```

Comment:
Here we should document that by default the network outbound configuration is through an Azure Load Balancer, and that it is immutable.